src/Compilers/Core/Portable/CommandLine/CommandLineArguments.cs

```diff
@@ -75,72 +77,70 @@ public abstract class CommandLineArguments
         /// <summary>
         /// Compilation name or null if not specified.
         /// </summary>
-        public string CompilationName { get; internal set; }
+        public string? CompilationName { get; internal set; }
```

Comment:
I distinctly remember fixing a bug that came about specifically because the code didn't realize `CompilationName` could be `null`. Seeing this makes me happy for the time I spent fixing that bug some years ago.